src/pages/basic_data/institution_admin/person_register/PersonTableRow.tsx

```diff
@@ -235,137 +242,139 @@ export const PersonTableRow = ({
                     )}
                   </Box>
                   <Divider flexItem orientation="vertical" />
-                  {values.employments.length > 0 && (
-                    <div>
-                      <Typography variant="overline" display="block" gutterBottom>
-                        {t('common.employments')}
-                      </Typography>
-                      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
-                        <Field name={`${employmentBaseFieldName}.organization`}>
-                          {({ field }: FieldProps<string>) => (
-                            <AffiliationHierarchy unitUri={field.value} commaSeparated />
-                          )}
-                        </Field>
-
-                        <Box display={{ display: 'flex', gap: '1rem' }}>
-                          <PositionField
-                            fieldName={`${employmentBaseFieldName}.type`}
-                            disabled={isSubmitting}
-                            includeDisabledPositions
-                          />
-
-                          <Field name={`${employmentBaseFieldName}.fullTimeEquivalentPercentage`}>
-                            {({ field, meta: { touched, error } }: FieldProps<string>) => (
-                              <TextField
-                                {...field}
-                                value={field.value ?? ''}
-                                required
-                                disabled={isSubmitting}
-                                fullWidth
-                                type="number"
-                                inputProps={{ min: '0', max: '100' }}
-                                variant="filled"
-                                label={t('basic_data.add_employee.position_percent')}
-                                error={touched && !!error}
-                                helperText={touched && error}
-                              />
+                  {isLoadingInstitutionUser ? (
+                    <CircularProgress sx={{ margin: 'auto' }} aria-labelledby="edit-person-label" />
+                  ) : (
+                    values.employments.length > 0 && (
+                      <div>
+                        <Typography variant="overline" display="block" gutterBottom>
```

Comment:
skjer det noe om vi fjerner block her? Ved første øyekast ser den overflødig ut for meg
```suggestion
                        <Typography variant="overline" gutterBottom>
```